src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -489,8 +489,39 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 			send_ack = true;
 		}
 
-	} else {
+	} else if (cmd_mavlink.command == MAV_CMD_DO_SET_ACTUATOR) {
+		// since we're only paying attention to 3 AUX outputs, the
```

Comment:
This also needs to be added to navigator.

Reply:
Yep, will do.

Reply:
To avoid code duplication, you could move it to the commander. This handles commands in a mission as well as MAVLink commands. Compare with the code and successful tests in https://github.com/PX4/PX4-Autopilot/pull/14809

Reply:
`since we're only paying attention to 3 AUX outputs` What is the rational behind this? This is very confusing, and is not obvious unless you dig into the code